azure/lerna-package-lock.json

```diff
@@ -671,25 +671,25 @@
       },
```

Comment:
Since your changes don't affect the packages in `azure` and `common`, we should probably revert the package-lock changes in those directories.